rclpy/src/rclpy/_rclpy.c

```diff
@@ -2654,13 +2655,13 @@ rclpy_wait_set_is_ready(PyObject * Py_UNUSED(self), PyObject * args)
     entities = (void *)wait_set->events;
     num_entities = wait_set->size_of_events;
   } else {
-    PyErr_Format(PyExc_RuntimeError,
+    PyErr_Format(RCLError,
       "'%s' is not a known entity", entity_type);
     return NULL;
   }
 
   if (!entities) {
-    PyErr_Format(PyExc_RuntimeError, "Wait set '%s' isn't allocated", entity_type);
+    PyErr_Format(RCLError, "Wait set '%s' isn't allocated", entity_type);
```

Comment:
I think this error is not because of an error in `rcl`.
```suggestion
    PyErr_Format(PyExc_RuntimeError, "Wait set '%s' isn't allocated", entity_type);
```